### ISSUE TITLE
Build fails on RHEL 5.8

### ISSUE THREAD

**Issue by ssttaarr** [yellow-apple/fusesniff#2]
```
[star@vm-tesco-db fusesniff]$ sh build.sh 
main.c: In function ‘sniff_getattr’:
main.c:90: warning: implicit declaration of function ‘fstatat’
main.c: In function ‘sniff_readdir’:
main.c:107: warning: implicit declaration of function ‘fdopendir’
main.c:107: warning: initialization makes pointer from integer without a cast
main.c:111: warning: suggest parentheses around assignment used as truth value
main.c: In function ‘sniff_opendir’:
main.c:122: warning: implicit declaration of function ‘openat’
main.c:122: error: ‘O_DIRECTORY’ undeclared (first use in this function)
main.c:122: error: (Each undeclared identifier is reported only once
main.c:122: error: for each function it appears in.)
main.c: In function ‘sniff_read’:
main.c:170: warning: implicit declaration of function ‘pread’
main.c: In function ‘sniff_write’:
main.c:181: warning: implicit declaration of function ‘pwrite’
main.c: In function ‘sniff_utimens’:
main.c:192: warning: implicit declaration of function ‘utimensat’
main.c: In function ‘sniff_unlink’:
main.c:203: warning: implicit declaration of function ‘unlinkat’
main.c: In function ‘sniff_rmdir’:
main.c:211: error: ‘AT_REMOVEDIR’ undeclared (first use in this function)
main.c: In function ‘sniff_rename’:
main.c:219: warning: implicit declaration of function ‘renameat’
main.c: In function ‘sniff_link’:
main.c:230: warning: implicit declaration of function ‘linkat’
main.c: In function ‘sniff_chmod’:
main.c:241: warning: implicit declaration of function ‘fchmodat’
main.c: In function ‘sniff_chown’:
main.c:252: warning: implicit declaration of function ‘fchownat’
main.c: In function ‘sniff_mknod’:
main.c:316: warning: implicit declaration of function ‘mknodat’
main.c: In function ‘sniff_mkdir’:
main.c:327: warning: implicit declaration of function ‘mkdirat’
main.c:327: error: ‘S_IFDIR’ undeclared (first use in this function)
main.c: In function ‘sniff_readlink’:
main.c:338: warning: implicit declaration of function ‘readlinkat’
main.c: In function ‘main’:
main.c:392: warning: suggest parentheses around && within ||
main.c:396: error: ‘O_DIRECTORY’ undeclared (first use in this function)
main.c:406:2: warning: no newline at end of file
[star@vm-tesco-db fusesniff]$ 
```

**Comment by yellow-apple** [comment]
Should be probably fixed with 8c1335b815f8b1109407c0a6285ee65ba935f2b6. Need confirmation.

**Comment by yellow-apple** [comment]
Confirmed.
